### PR TITLE
Export all types from index.d.ts

### PR DESCRIPTION
**Pull request by mocca102**
Instead of importing types from this path `@constructor-io/constructorio-client-javascript/lib/types ` We will be able to import types from this path `@constructor-io/constructorio-client-javascript`

### CODE REVIEW

**Comment by esezen** [comment]
Would this break anything for projects using `@constructor-io/constructorio-client-javascript/lib/types` at the moment or would it be supporting both `@constructor-io/constructorio-client-javascript/lib/types` and `@constructor-io/constructorio-client-javascript`?

**Comment by mocca102** [comment]
> Would this break anything for projects using `@constructor-io/constructorio-client-javascript/lib/types` at the moment or would it be supporting both `@constructor-io/constructorio-client-javascript/lib/types` and `@constructor-io/constructorio-client-javascript`?

It should support both. This screenshot for autocomplete-os-ui linking to this PR 
Screenshot 2023-09-12 at 8 44 17 PM

**Comment by mocca102** [comment]
@esezen Made it backward compatible

**Review by esezen — COMMENTED** [review]
I tested this by creating a new project and using this PR to install the package. It appears like both old and new way work as expected. Let's get one more review from the team before we merge but this is looking great! We might want to do the same for Node

**Review by stanlp1 — APPROVED** [review]
Tested in a new project, also seems to work as expected.

**Pull request merged**
